client/my-sites/plans-v2/plans-column/index.tsx

```diff
@@ -70,15 +70,17 @@ const PlanComponent = ( {
 				: 'jetpack_security_daily_monthly';
 
 		const upgradeToProductSlug = productSlug && getRealtimeFromDaily( productSlug );
+		const item = upgradeToProductSlug && slugToItem( upgradeToProductSlug );
+		const selectorProductToUpgrade = item && itemToSelectorProduct( item );
```

Comment:
Can probably be updated to `slugToSelectorProduct` now 😄

Reply:
Yup! Updating...